ch_pipeline/hfb/analysis.py

```diff
@@ -195,6 +195,70 @@ def process(self, stream):
         return out
 
 
+class MakeHighFreqResRingMap(task.SingleTask):
+    """Combine frequency and sub-frequency axes and reorder axes for HFB ringmaps"""
+
+    def process(self, stream):
+        """Convert HFBRingMap to HFBHighResRingMap container
+
+        Parameters
+        ----------
+        stream : HFBRingMap
+            Ringmap with frequency and subfrequency axis
+
+        Returns
+        -------
+        out : HFBHighResRingMap
+            Ringmap with single high-resolution frequency axis
+        """
+
+        # Change distributed axis from freq to el
+        stream.redistribute("el")
+
+        # Retrieve data and weights
+        data = stream.hfb[:]
+        weight = stream.weight[:]
+
+        # Make combined frequency axis
+        cfreq = stream.index_map["freq"]["centre"]
+        subfreq = stream.index_map["subfreq"]
+        freq = (cfreq[:, np.newaxis] + subfreq).flatten()
+
+        # Retrieve beam, elevation, and RA axes
+        beam = stream.index_map["beam"]
+        el = stream.el
+        ra = stream.ra
+
+        # Move frequency and subfrequency axes to the back
+        axes = (2, 3, 4, 0, 1)
+        data = data.transpose(axes)
+        weight = weight.transpose(axes)
+
+        # Determine new shape of data with combined frequency axis,
+        # giving a length of None to the (distributed) el axis
+        nbeam = len(beam)
+        nra = len(ra)
+        nfreq = len(cfreq)
+        nsubfreq = len(subfreq)
+        new_shape = (nbeam, None, nra, nfreq * nsubfreq)
+
+        # Combine frequency and sub-frequency axes
+        data = data.reshape(new_shape)
+        weight = weight.reshape(new_shape)
+
+        # Create container to hold output
+        out = containers.HFBHighResRingMap(
+            beam=beam, el=el, ra=ra, freq=freq, attrs_from=stream
```

Comment:
Why not use `axes_from`? That will just copy over the el and ra axes.

Reply:
Right, that fix ended up in a different branch. I added it here now.